openlabcmd/openlabcmd/zk.py

```diff
@@ -29,6 +29,7 @@
     'github_user_token': None,
     'heartbeat_timeout_second': 600,
     'logging_level': 'DEBUG',
+    'service_count_allow_to_raise': 3,
```

Comment:
`service_restart_max_allow_times` is better.

Reply:
or just `service_restart_max_times`

Reply:
done